components/src/main/java/uk/gov/hmrc/components/organism/menu/MenuPanelRowView.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.components.organism.menu
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.RelativeLayout
+import uk.gov.hmrc.components.R
+import uk.gov.hmrc.components.base.PaddedComponent
+import uk.gov.hmrc.components.databinding.ComponentMenuPanelRowBinding
+import uk.gov.hmrc.components.extensions.addRipple
+import uk.gov.hmrc.components.extensions.setMargins
+
+class MenuPanelRowView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : RelativeLayout(context, attrs, defStyleAttr), PaddedComponent {
```

Comment:
Could this extend MaterialCardView rather than RelativeLayout? Then we wouldn't need it in the xml.